backend/organization/views/project_views.py

```diff
@@ -173,9 +175,31 @@ def post(self, request):
         except ProjectStatus.DoesNotExist:
             return Response({
                 'message': "Passed status {} does not exist".format(request.data["status"])
-            })
+            })  
 
-        project = create_new_project(request.data)
+        try:
+            translations = get_project_translations(request)            
+        except ValueError:
+            return Response({
+                'message': "Please use the same language for all texts."
+            }, status=status.HTTP_400_BAD_REQUEST)  
+
+        project = create_new_project(request.data, translations['source_language'])
+
+        for language in translations:
+            if not language == translations['source_language']:
+                texts = translations[language]
+                translation = ProjectTranslation.objects.create(
+                    project=project, 
+                    language=language,
```

Comment:
Same here, language is a foreign key so you have first get the language object before you create project translation.